.github/workflows/check-i18n-changes.yml

```
@@ -0,0 +1,23 @@
+name: Protect content from manual changes
+
+on: 
+  pull_request:
+    paths:
+    - 'content/**'
+
+jobs:
+  creator-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: 
+        if: github.actor != "nodejs-crowdin" || github.actor != "github-actions"
+        uses: thollander/actions-comment-pull-request@master
```

Should pin to a specific version. This is the only one I saw on the Marketplace for this action
```suggestion
        uses: thollander/actions-comment-pull-request@1.0.0
```